src/mechanics/cell/Cell.h

```diff
@@ -1,5 +1,6 @@
 #pragma once
 
+#include "base/UniqueId.h"
```

Comment:
No longer needed, now that you supply the id via the constructor.

Reply:
Good point. I ll remove it